TestAssets/TestProjects/TestAppSimple/project.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.0-rc2-*"
```

Comment:
That's what `dotnet new` generates, and it's what makes the app "portable".

Without it and without a "runtimes" section, your app isn't really either of the 2 app models we support: portable or standalone.